### DISCUSSION TITLE
BUG in Web-User Interface

### DISCUSSION THREAD

**Discussion by ghost** [coqui-ai/TTS#828] [Archived]
This is the output when I am running on my local machine .(current version = v0.3.1)
![image](https://user-images.githubusercontent.com/88183889/134165962-f8e972ac-9f6b-422e-b2cc-bb3521199d57.png)

 but In actual documentation, they provide drop-down menu to choose models there 
![image](https://user-images.githubusercontent.com/88183889/134166294-cedbe7da-1d54-4ad1-876d-b29782aa8ab1.png)

I would like to fix  this bug in your current released version and for my local machine. Can anyone guide me which file is responsible for the WEB-UI.. Kindly provide the file name and which directory the file is present...
I have tried in server directory but it's not working.

**Comment by ghost** [comment]
Actual Image when I work in local machine:
![image](https://user-images.githubusercontent.com/88183889/134265962-784df825-9be2-43de-88a4-db082b38f293.png)


But it is supposed to be like this 
![image](https://user-images.githubusercontent.com/88183889/134265933-d13ce3dc-416b-4afe-a688-a568f1227ee4.png)

so kindly take a look here and solve this

**Comment by erogol** [comment]
you already created an issue for this.

**Reply by ghost** [reply]
@erogol  sorry sir
